hedera-node/src/test/java/com/hedera/services/store/TypedTokenStoreTest.java

```diff
@@ -297,7 +300,6 @@ void savesTokenAsExpected() {
 		verify(uniqueTokens).remove(expectedPastUniqTokenId);
 		verify(uniqueTokenAssociations).associate(new EntityId(modelToken.getId()), expectedNewUniqTokenId);
 		verify(uniqueTokenAssociations).disassociate(new EntityId(modelToken.getId()), expectedPastUniqTokenId);
-		verify(uniqueTokenOwnerships).associate(treasuryId, expectedNewUniqTokenId);
```

Comment:
We have to verify that the new FCOTMR was called with associate. In general we should have code coverage on the new feature

Reply:
This is never called since during minting op we do association for uniqueTreasuryOwnership FCOTMR and not for uniqueTokenOwnerships